addons/skin.confluence/720p/MyPVRRecordings.xml

```diff
@@ -296,28 +294,23 @@
 					<left>10</left>
 					<top>548</top>
 					<width>290</width>
-					<height>12</height>>
-					<info>PVR.backenddiskspaceprogr</info>
-					<visible>!IntegerGreaterThan(PVR.backenddiskspaceprogr,100)</visible>
-				</control>
+					<height>12</height>&gt;
```

Comment:
nice, thanks. :) (wondering a bit why the XMLParser didnt throw any mistake / warning)